spec/sub_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'array'
+describe Array do 
+	let (:array) {[1,2,3,4,5]}
+
+	    add = Proc.new{|n, i| n + i}
+		substract = Proc.new{|n, i| n - i}	
+		multiply = Proc.new{|n, i| n * i}
+	    divide = Proc.new{|n, i| n / i}
+
+	    
+
+	      it "it returns 15 when we concenate each element" do
+	      expect(array.inject_1 &add).to eq(15)
+	  end
```

Comment:
Lint/BlockAlignment: `end` at 14, 3 is not aligned with `it "it returns 15 when we concenate each element" do` at 12, 7